src/index.json

```diff
@@ -8841,6 +8841,64 @@
             }
         ],
         "arcdata": [
+            {
+                "downloadUrl": "https://arcdataazurecliextension.blob.core.windows.net/stage/arcdata-1.4.8-py2.py3-none-any.whl",
```

Comment:
why the host name changed?

Reply:
> why the host name changed?

@kairu-ms  - Changed host  to production location. File will be available on release day.